source/MaterialXGenShader/UnitSystem.h

```diff
@@ -14,7 +14,7 @@
 #include <MaterialXGenShader/ShaderNode.h>
 #include <MaterialXGenShader/ShaderNodeImpl.h>
 #include <MaterialXGenShader/TypeDesc.h>
-#include <MaterialXGenShader/UnitConverter.h>
+#include <MaterialXCore/UnitConverter.h>
 
 #include <MaterialXCore/Document.h>
```

Comment:
Can't seem to put a comment there but in the create() method a language is a required input. It's a cached value that only seems to be used for getting the implementation. I think that was one reason why it was in GenShader. I think the easiest thing to do is to not cache this but require it as an input argument to UnitSystem::getImplementationName().

Reply:
That’s right, the only use is in `UnitSystem::getImplementationName()`. But `getImplementationName` is in turn called by `UnitSystem::supportsTransform()` and `UnitSystem::createNode()`, which would also have to take the language name as argument.

But I’m not sure I see the relevance? We are leaving `UnitSystem` in GenShader; only `UnitConverter` is moving to Core.

Reply:
Yes, your right there is nothing that is required here now.